### PR TITLE
KAFKA-10557: Missing docs when describing topic configs including

### PR DESCRIPTION
**Pull request by edoardocomar**
documentation

Added unit test

Co-authored-by: Edoardo Comar <ecomar@uk.ibm.com>
Co-authored-by: Mickael Maison <mickael.maison@gmail.com>

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by edoardocomar** [comment]
@rajinisivaram may you take a look please ?

File: core/src/test/scala/unit/kafka/server/AdminManagerTest.scala [hunk 1]
```diff
@@ -80,4 +81,30 @@ class AdminManagerTest {
     assertEquals(Errors.NONE.code, results.head.errorCode())
     assertFalse("Should return configs", results.head.configs().isEmpty)
   }
+
+  @Test
+  def testDescribeConfigsWithDocumentation(): Unit = {
+    EasyMock.expect(zkClient.getEntityConfigs(ConfigType.Topic, topic)).andReturn(new java.util.Properties)
+    EasyMock.expect(zkClient.getEntityConfigs(ConfigType.Broker, brokerId.toString)).andReturn(TestUtils.createBrokerConfig(brokerId,"zk"))
```
Comment (rajinisivaram) [inline core/src/test/scala/unit/kafka/server/AdminManagerTest.scala]: nit: space before "zk" - why is broker id anyway :-) ?

File: core/src/test/scala/unit/kafka/server/AdminManagerTest.scala [hunk 2]
```diff
@@ -80,4 +81,30 @@ class AdminManagerTest {
...
+      new DescribeConfigsRequestData.DescribeConfigsResource()
+        .setResourceName(brokerId.toString)
+        .setResourceType(ConfigResource.Type.BROKER.id))
+
+    val results: List[DescribeConfigsResponseData.DescribeConfigsResult] = adminManager.describeConfigs(resources, true, true)
+    assertEquals(2, results.size)
+    results.foreach(r => {
+      assertEquals(Errors.NONE.code, r.errorCode)
+      assertFalse("Should return configs", r.configs.isEmpty)
+      r.configs.forEach(c => assertNotNull(s"Config ${c.name} should have documentation", c.documentation))
```
Comment (rajinisivaram) [inline core/src/test/scala/unit/kafka/server/AdminManagerTest.scala]: we could check documentation is not empty as well?

**Review by rajinisivaram — APPROVED** [review]
@edoardocomar @mimaison Thanks for the PR. Left a couple of minor comments, apart from those LGTM. Approved so that @mimaison can merge it, another review not required.

File: core/src/test/scala/unit/kafka/server/AdminManagerTest.scala (hunk 2 above)
Comment (edoardocomar) [reply core/src/test/scala/unit/kafka/server/AdminManagerTest.scala]: sure thanks

**Comment by mimaison** [comment]
Thanks @rajinisivaram for the review.
Test failures are not related, merging.

**Comment by mimaison** [comment]
I'll also merge into 2.6

**Pull request merged**